packages/hash/frontend/src/theme/components/utils/MuiCssBaselineThemeOptions.ts

```diff
@@ -241,10 +241,6 @@ export const MuiCssBaselineThemeOptions: Components["MuiCssBaseline"] = {
 
           a {
             text-decoration: none;
-
-            &:visited {
-              color: initial;
-            }
```

Comment:
removed this because it kept messing up with the styling when `Button` component is rendered as an `a` tag